test/error.js

```diff
@@ -124,26 +127,34 @@ test('result.killed is false on process error', async t => {
 
 test('result.killed is false on process error, in sync mode', t => {
 	const {killed} = t.throws(() => {
-		execa.sync('wrong command');
+		execaSync('wrong command');
 	});
 	t.false(killed);
 });
 
 if (process.platform === 'darwin') {
-	test.cb('sanity check: child_process.exec also has killed.false if killed indirectly', t => {
-		const {pid} = childProcess.exec('noop', error => {
-			t.truthy(error);
-			t.false(error.killed);
-			t.end();
+	test('sanity check: child_process.exec also has killed.false if killed indirectly', async t => {
+		const promise = new Promise((resolve, reject) => {
+			const {pid} = childProcess.exec('noop', error => {
```

Comment:
If we use promises, we might want to use `const pExec = utils.promisify(childProcess.exec)` instead of `new Promise`.

Reply:
Thanks for pointing it out! Thanks to you, I was able to spot a bug - it's meant to say `'noop.js'`. I'll make sure to search for all references to the new `.js` fixtures and ensure they have an extension.